client/src/components/game/phases/Events.vue

```diff
@@ -54,19 +55,18 @@ export default class Events extends Vue {
     return this.$tstore.state.marsEventsProcessed;
   }
 
+  get eventNumber(): number {
+    const eventNumber = this.$tstore.state.marsEventsProcessed + 1;
+    return eventNumber;
+  }
+
   get currentEvent(): MarsEventData {
```

Comment:
Cool, for a future refactor I think we can use https://vuex.vuejs.org/guide/getters.html#the-mapgetters-helper to map `currentEvent` to a property on this Vue component

Reply:
`mapgetters` isn't typed so I would rather not use it because we won't know if something breaks. Something like this https://github.com/championswimmer/vuex-module-decorators would also work.